prime-angular-frontend/src/app/shared/components/forms/address/address.component.html

```diff
@@ -1,7 +1,12 @@
 <app-address-autocomplete [bcOnly]="bcOnly"
                           (autocompleteAddress)="onAutocomplete($event)"></app-address-autocomplete>
 
-<div [formGroup]="form">
+<button *ngIf="simpleMode"
```

Comment:
> In the mockups I thought this would still be visible after clicking it, not sure if this is a change or we are allowed to toggle it back and forth.

Good point. I didn't know the mockup is interactive. Will make the button greyed out after entering manual mode. But not sure if it's toggle button.